service/relayer/Dockerfile

```
@@ -14,15 +14,30 @@ RUN \
+COPY service/relayer/redstone-bridge/package*.json ./
+RUN npm install
+
+COPY service/relayer/redstone-bridge/tsconfig.json ./
+COPY service/relayer/redstone-bridge/src ./src
+RUN npm run build
+
 # Runtime image
-FROM debian:12-slim
+FROM node:24-slim
 
 RUN apt-get update && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)